stripecardscan/src/main/java/com/stripe/android/stripecardscan/cardscan/CardScanSheet.kt

```diff
@@ -43,34 +43,53 @@ private const val CARD_SCAN_FRAGMENT_TAG = "CardScanFragmentTag"
 
 class CardScanSheet private constructor(private val stripePublishableKey: String) {
 
-    private var onFinished:
-        ((cardImageVerificationSheetResult: CardScanSheetResult) -> Unit)? = null
     private lateinit var launcher: ActivityResultLauncher<CardScanSheetParams>
 
+    /**
+     * Callback to notify when scanning finishes and a result is available.
+     */
+    fun interface CardScanResultCallback {
```

Comment:
This is added to ease Java integration. There is a similar callback for payment at  [PaymentResultCallback](https://github.com/stripe/stripe-android/blob/3e92b79190834dc3aab1c2d9ac2dfb7bc343afd2/payments-core/src/main/java/com/stripe/android/payments/paymentlauncher/PaymentLauncher.kt#L41)